### ISSUE TITLE
"Hide time picker" also hides refresh button

### ISSUE THREAD

**Issue by hahnjo** [grafana/grafana#11782]
### What Grafana version are you using?
v5.1.0

### What datasource are you using?
InfluxDB

### What OS are you running grafana on?
CentOS 7

### What did you do?
Tried to hide the time picker

### What was the expected result?
The users shouldn't be able to select a time range because that dashboard only uses panels with time overwrite and shows the current value. Refreshing makes sense because the current data changes by definition.

### What happened instead?
The refresh button is also hidden

### Use case
Having a dashboard showing the current status, maybe an average over the last hour.

**Comment by daniellee** [comment]
Before hiding the time picker, you could set an auto refresh interval. Would that work in your case?

Otherwise, hiding the time picker but showing the refresh icon feels like an unusual use case that could be quite confusing for most users.

**Comment by hahnjo** [comment]
Enabling auto refresh would probably work, but I think you can't see it either if you "hide time picker". Plus this puts a constant load on the database as long as the tab is opened which we usually want to avoid.

I agree that there are cases where you don't want to have either, for example the default home dashboard. Would it be acceptable to have an extra checkbox to "show refresh button" (open to improvements on naming this)?

**Comment by scaarup** [comment]
+1
Visualising data from MySQL in a table. Not time related at alle, but data changes every ~10 minute. The refresh button should be there, but  timer-picker is totally irrelevant.

**Comment by Cossey** [comment]
+1
Have a few dashboards with all fixed time ranges like the other two above; no need for Time Range, but would be nice to be able to toggle refreshing. Fortunately in my case it isn't critical and I can just set a auto-refresh time before hiding the button or via the JSON markup.

Just curious as to why having more options is bad?

**Comment by JoeeGrigg** [comment]
I also agree. I have a few dashboards that have either fixed time rangers or basic counts of data that do not take into account the time range. It would be nice to allow the user of the dashboard to periodically refresh the data, when they want the updated numbers, without them possibly being confused over the time range that is auto selected and never used.

I would also be curious to understand why options are bad? Yes if the options are getting confusing but if it is a checkbox saying "Hide Refresh Button" I don't really see how that could possibly be confusing in any way.

**Comment by jatinrai199** [comment]
+1
no need for Time Range, but it would be nice to be use auto refresh controls. both time range and auto-refresh should be treated as separate usecases.

**Comment by reuvygroovy** [comment]
are there any plans to include this, maybe in 7?

**Comment by franck102** [comment]
This issue and the related feature request are both closed, any chance of seeing this implemented?
The refresh button is completely unrelated to hiding the time picker: we absolutely need to hide the time picker because some of the panels on the page won't take it into account, and this leaves the user with no means to refresh the data.

**Comment by RhysDavis** [comment]
I also think it's strange to have hiding the time picker also hide the auto refresh. I have a dashboard of universal lifetime stats that changes constantly and need the auto refresh functionality but not the time picker functionality. 
Would love to see this implemented

**Comment by GnarTech** [comment]
This needs to be reopened. I especially wanted this for our homepage and would love to see a checkbox option to still show refresh wheel. Users are refreshing the browser to bypass this. Also worth mentioning, if I try to set an auto-refresh before saving, it gives me an error when I try to save. Haven't investigated further at the moment. Just noticed.

**Comment by digrich** [comment]
We do have a use case of showing the customer summary in the custom panel which is not using dashboard time range but we do want an auto-refresh option to keep updating the data on the dashboard.  

In many use-cases, we don't want the user to use a dashboard time picker but at the same time we want them to choose a refresh interval to keep updating the data.

**Comment by mertbakir** [comment]
I have similar use case, I don't want users to interact with time picker but keep the refresh button and auto-refresh options.

**Comment by wz2b** [comment]
This issue is pretty old, but I have this problem, too.  I'd like to enable refresh but not the time interval for dashboards that always only show the latest state data.

**Comment by david-proctor** [comment]
+1. I have a use case where I am showing latest state from a database query, and it looks like I am not alone.

**Comment by podolsky** [comment]
+1 here as well with the same issue - I want the user to be able to potentially set a range of refresh rates (including no refresh) for  panels that have no time-picker-time-range dependency, and displaying a time range in the time picker is just confusing to users. 

Also there's no ability to work around it either by putting different datalinks to the same dashboard, each corresponding to setting `&refresh=` to different values in the URL… the first time one of the datalinks is clicked it will update the refresh properly and activate refreshes in panels, but subsequent changes without hard-reloading the page will update the URL's refresh value but not actually affect the refresh rate.

**Comment by pcroc** [comment]
I find this issue quite frustrating. We have a daily dashboard which we review, as a team, every day. Folks usually keep it open all the time in a background browser tab. When I switch to it, the only way to know it is up to date is to refresh the entire browser page. There are a lot of panels showing long term trends, so we don't want it to automatically refresh at short intervals.

The refresh button and the time picker are completely separate controls, I don't understand the reasoning for hiding the refresh button just because the time picker is hidden.

**Comment by dprokop** [comment]
This will be fixed with this community contribution: https://github.com/grafana/grafana/pull/83697

**Comment by jacquesnguyen** [comment]
Can this be reopened ?
There are definitely use cases where you don't need the time picker but you need the refresh button. Would be very useful.
Furthermore, I find that having the refresh button is always useful, as sometimes, some panels don't refresh entirely on variable changes.
